en/core-libraries/components/pagination.rst

```diff
@@ -90,6 +90,11 @@ array after the model you wish to configure::
 The values of the ``Post`` and ``Author`` keys could contain all the properties
 that a model/key less ``$paginate`` array could.
 
+.. note::
+    The ``$paginate`` variable is not automatically passed to the :php:class:`PaginatorComponent`'s settings. You must explicitly set the :php:class:`PaginatorComponent`'s settings using the following line::
+
+        $this->Paginator->settings = $this->paginate;
```

Comment:
Your solution `$this->Paginator->settings = $this->paginate` makes you lose all settings you directly set on the component.
So I think this might make more people trip than before.

I still think making the difference between fully non component and fully component usage more clear would be the best way to go.